ethstats/ethstats.go

```diff
@@ -96,14 +96,14 @@ type blockChain interface {
 // Service implements an Ethereum netstats reporting daemon that pushes local
 // chain statistics up to a monitoring server.
 type Service struct {
-	server  *p2p.Server              // Peer-to-peer server to retrieve networking infos
-	eth     *eth.Ethereum            // Full Ethereum service if monitoring a full node
-	les     *les.LightEthereum       // Light Ethereum service if monitoring a light node
-	engine  consensus.Engine         // Consensus engine to retrieve variadic block fields
-	backend *istanbulBackend.Backend // Istanbul consensus backend
-
-	node string // Name of the node to display on the monitoring page
-	host string // Remote address of the monitoring service
+	server    *p2p.Server              // Peer-to-peer server to retrieve networking infos
+	eth       *eth.Ethereum            // Full Ethereum service if monitoring a full node
+	les       *les.LightEthereum       // Light Ethereum service if monitoring a light node
+	engine    consensus.Engine         // Consensus engine to retrieve variadic block fields
+	backend   *istanbulBackend.Backend // Istanbul consensus backend
+	etherBase common.Address
```

Comment:
Add comment?
Also cannot see where this variable is used...

Reply:
it's used here: https://github.com/celo-org/celo-blockchain/pull/677/files/71e60d3c806c1555d566bd7a4e3f212c721b894d#diff-9d0b059013dd700f399bf16ba37421afR123